core/templates/dev/head/css/oppia.css

```diff
@@ -2924,3 +2924,6 @@ md-card.preview-conversation-skin-supplemental-card {
   color: red;
   margin-bottom: 10px;
 }
+/* Icon fonts size definition */
+.material-icons.md-100 { font-size: 100px; }
```

Comment:
Please put this with the other similar size definitions in this file, and follow the existing style (newline after '{', etc.)